src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/AccountsChannelMock.kt

```diff
@@ -12,7 +12,7 @@ internal class AccountsChannelMock {
           "type": "subscribed",
           "connection_id": "c49e6d6b-d489-4140-bbaf-dc11b2fab13a",
           "message_id": 1,
-          "channel": "v3_accounts",
+          "channel": "v4_subaccounts",
```

Comment:
I think there is a lot more refactoring to be done here 😭 

This payload is from `v3_accounts` channel on v3 api. We have to pretty much overhaul all of our mock data that pertains to v3

Reply:
This is true for all other payloads in these files as well.

Reply:
I know.. But that's too much work for me 😸    Let's land this for now and fix them later.

Those mocks are used everywhere and contain too much irrelevant information.  It's better to move away from using them in the future.

Reply:
Sounds good 👍